src/components/DeleteConfirmation/DeleteConfirmation.js

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import "./DeleteConfirmation.scss";
+
+function DeleteConfirmation(props) {
+  return (
+    <div className="confirmation-background">
+      <div className="delete-note">Delete Lesson: {props.lessonName}?</div>
+      <div className="delete-lesson-button-container">
+        <div
+          className="delete-confirmation-button"
+          onClick={() => props.handleDelete()}
```

Comment:
`onclick={handleDelete}`

Reply:
also addressed in other branch

Reply:
since i pulled the other branch, its also on this one